src/components/SearchResult.tsx

```diff
@@ -63,11 +69,20 @@ export interface SearchResultProps {
 
 class SearchResult extends React.Component<SearchResultProps & WithStyles<typeof styles>> {
     render() {
-        const { classes, iconColor, iconSrc, iconType, label, to } = this.props;
+        const { classes, iconColor, iconSrc, iconType, query, label, to } = this.props;
 
         const isAvatar = iconType === 'avatar';
         const isIcon = iconType === 'icon';
 
+        const regex = new RegExp('(' + query + ')', 'gi');
+        const highlightedLabel = (
+          <Typography noWrap>
+              { label.split(regex).map((part, i) =>
+                <span key={i} style={part.toLowerCase() === query ? { fontWeight: 'bold'} : {}}>{part}</span>
```

Comment:
nit: ik zou er een `className` van maken, dan kunnen we later bijvoorbeeld ook de kleur aanpassen.

```
{ label.split(regex).map((part, i) => {
    const className = part.toLowerCase() === query ? 'highlight'
                                                   : undefined;

    return <span key={i} className={className}>{part}</span>;
}) }
```